### PR TITLE
fix CODEOWNERS

### PR DESCRIPTION
**Pull request by kaworu**
https://github.com/cilium/cilium/pull/18972 introduced changes that broke the CODEOWNERS linter.

### CODE REVIEW

**Review by qmonnet — APPROVED** [review]

**Review by pchaigno — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Comment by kaworu** [comment]
Wish to have https://github.com/cilium/cilium/pull/18981 merged first, then this PR rebased on top before merging.

**Comment by qmonnet** [comment]
We should be able to discard the first commit (the revert) from this PR after my PR is merged.

**Review by qmonnet — APPROVED** [review]

**Comment by kaworu** [comment]
[CODEOWNERS checks passed](https://github.com/cilium/cilium/runs/5378354449?check_suite_focus=true), marking as `ready-to-merge`.

**Review by joestringer — APPROVED** [review]

**Pull request merged**
